distribution/ddf-common/src/main/resources-filtered/etc/application-definitions/security-services.json

```diff
@@ -20,6 +20,7 @@
     "mvn:ddf.platform.security/security-oidc-realm/${project.version}",
     "mvn:ddf.platform.security/security-guest-realm/${project.version}",
```

Comment:
This should be removed. Does the oidc one also need fixed? It looks like we changed the realm artifact ids

Reply:
Yes OIDC needs to change too